src/spacel/model/app.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, orbit, params=None):
         self.scheme = params.get('scheme', 'internet-facing')
         self.health_check = params.get('health_check', 'TCP:80')
         self.local_health_check = params.get('health_check', 'TCP:80')
+        self.loadbalancer = params.get('loadbalancer', 'true')
```

Comment:
Maybe turn this into a `bool` right away? `model.*` can make the JSON as pythonic as possible.

(Yes I keep adding passthrough maps that go against this idea, start yelling at me for that).